evm/vm/forks/sharding/validation.py

```diff
@@ -0,0 +1,14 @@
+from evm.exceptions import (
+    ValidationError,
+)
+
+from evm.vm.forks.frontier.validation import (
+    validate_frontier_transaction,
+)
+
+
+def validate_sharding_transaction(evm, transaction):
+    # TODO:Update transaction validation logic in Sharding
+    # e.g. checking shard_id < SHARD_COUNT
+
+    validate_frontier_transaction(evm, transaction)
```

Comment:
Add a comment for using `validate_frontier_transaction` but `validate_homestead_transaction` on purpose would lesser ambiguity. (`validate_homestead_transaction` doesn't apply for sharding)